sig-node/CONTRIBUTING.md

```diff
@@ -2,17 +2,90 @@
 
 Welcome!
 
-## For Kubernetes Contributions
+Thank you for your interest in contributing to SIG Node. SIG Node is one of the biggest SIGs in Kubernetes.
+Reliability and performance of millions of nodes running critical applications in production rely on a quality of your contribution.
+The diversity of workloads, environments, and scale SIG Node needs to support make every code change risky as every side effect needs to be evaluated.
+And the contribution is and realistically can be only evaluated by a small set of maintainers.
+
+Please make sure you understand and accept the challenge. Contributing instructions are designed to help you.
+
+## For Kubernetes Code Contributions
 
 Read the [Kubernetes Contributor Guide](https://github.com/kubernetes/community/tree/master/contributors/guide#contributor-guide).
 
 If you aspire to grow scope in the SIG, please review the [SIG Node contributor ladder](./sig-node-contributor-ladder.md) for SIG specific guidance.
 
 ### For Enhancements
 
-SIG Node enhancements are available in the <https://github.com/kubernetes/enhancements/tree/master/keps/sig-node>.
+Find out if [your thing is an enhancement a.k.a KEP](https://github.com/kubernetes/enhancements/?tab=readme-ov-file#is-my-thing-an-enhancement).
```

Comment:
A good way to do that is to open and issue and get feedback from node reviewers and approvers. You can even come present your idea at a weekly sig-node meeting.

Reply:
```suggestion
Find out if [your thing is an enhancement a.k.a KEP](https://github.com/kubernetes/enhancements/?tab=readme-ov-file#is-my-thing-an-enhancement). A good way to do that is to open and issue and get feedback from node reviewers and approvers. You can even come present your idea at a weekly sig-node meeting.
```